src/TransformableImage.php

```diff
@@ -149,6 +176,16 @@ private function cropImage(object $cropperData)
         $this->image->crop($cropperData->width, $cropperData->height, $cropperData->x, $cropperData->y);
     }
 
+    /**
+     * Orientate the image.
+     *
+     * @return void
+     */
+    private function orientateImage()
```

Comment:
Can you please position this method at the very end of the file?
And you can indeed update the documentation to add an example and the optional PHP library.

Reply:
Hello @ctessier, as you requested I moved the method to the bottom of the file and renamed the public method to autoOrientate.
I also added more details in the method comment, as well as in the requirements section of the readme.

Thank you for the review.